src/Pages/Login/Login.js

```diff
@@ -2,83 +2,99 @@ import React, { Component } from "react";
 import "./Login.scss";
 
 class Login extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            email: "",
-            pw: ""
-        };
-    }
-
-    hadleValueID = (e) =>  {
-        this.setState ({
-            email: e.target.value,
-        })
-    }
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      pw: "",
+    };
+  }
 
-    hadleValuePW = (e) => {
-        this.setState ({
-            pw : e.target.value
-        });
-    }
+  hadleValueID = (e) => {
+    this.setState({
+      email: e.target.value,
+    });
+  };
 
+  hadleValuePW = (e) => {
+    this.setState({
+      pw: e.target.value,
+    });
+  };
 
-    // 로그인용 버튼
-    hadleBtn = (e) => {
-        fetch("http://10.58.5.36:8000/user/sign-in", {
-            method: 'POST',
-            body: JSON.stringify({
-                email: this.state.email,
-                password: this.state.pw
-            })})
-            .then(res => res.json())
-            .then(res => {
-                if (res.token) { 
-                    localStorage.setItem('aesopToken', res.token);
-                    this.props.history.push("/main")
-                    console.log(res.token)
-                } 
-            })
-        
-    }
-    
+  // 로그인용 버튼
+  hadleBtn = (e) => {
+    fetch("http://10.58.7.53:8000/user/sign-in", {
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.pw,
+      }),
```

Comment:
```suggestion
  hadleBtn = (e) => {
    const { email, password } = this.state
    fetch("http://10.58.7.53:8000/user/sign-in", {
      method: "POST",
      body: JSON.stringify({ email, password }),
```
객체 단축 속성명을 활용하면 이렇게 사용할 수 있습니다